webknossos/webknossos/dataset/dataset.py

```diff
@@ -103,6 +103,8 @@
     + r"(?P<organization_id>[^/]*)/(?P<dataset_name>[^/]*)(/(view)?)?"
     + r"(\?token=(?P<sharing_token>[^#]*))?"
 )
+_ALLOWED_LAYER_NAME_REGEX = re.compile(r"^[A-Za-z0-9_\-]+[A-Za-z0-9_\-\.]*$")
```

Comment:
Please add a comment explaing this REGEX. Something like in your PR description. Which characters are actually allowed

Reply:
Done.